backend/FwLite/LocalWebApp/Routes/ProjectRoutes.cs

```diff
@@ -77,16 +80,13 @@ public static IEndpointConventionBuilder MapProjectRoutes(this WebApplication ap
                 SyncService syncService,
                 IOptions<AuthConfig> options,
                 CurrentProjectService currentProjectService,
-                string serverAuthority) =>
+                string serverAuthority,
+                [FromQuery] Guid lexboxProjectId) =>
```

Comment:
Should we verify that the ID points to a valid project before we store it?
On the other hand, the project could get deleted in Lexbox 2 minutes later and we need to be able to handle that 🤔

Reply:
Good point, it does kinda validate it during the sync (it'll fail), but I should probably clear the server if the initial upload fails. As for the GUID, it only really matters if the server is set anyway.